kuvaselaamo/settings.py

```diff
@@ -43,6 +44,9 @@
 
 ALLOWED_HOSTS = env.list("ALLOWED_HOSTS")
 
+if env("DEFAULT_FROM_EMAIL"):
```

Comment:
Since a type is and default for the env variable are provided (`str` and `helsinkikuvia@example.fi`). You can remove the if-clause and just do 

```python
DEFAULT_FROM_EMAIL = env("DEFAULT_FROM_EMAIL")
```

Reply:
Fixed.